Assets/Scripts/Editor/RoadDrawerTest.cs

```diff
@@ -45,10 +46,13 @@ public void Setup()
             nodes[0].connectedNodes = new int[] { 1 };
             nodes[1].connectedNodes = new int[] { 0 };
 
-            var roadDrawer = _container.Resolve<RoadDrawer>();
-            GameObject[] gameObjects = roadDrawer.SetupRoadSegments(nodes);
-            AssertThatRoadSegmentIsCorrect(gameObjects[0], "DeadEnd(Clone)", new Vector3(0, 0, 0), Direction.East);
-            AssertThatRoadSegmentIsCorrect(gameObjects[1], "DeadEnd(Clone)", new Vector3(1, 0, 0), Direction.West);
+			var roadDrawer = _container.Resolve<RoadDrawer>();
+            var roadBuilder = _container.Resolve<RoadBuilder>();
+			RoadSegment[] roadSegments = roadBuilder.CreateRoadSegments(nodes);
+            GameObject[] gameObjects = roadDrawer.DrawRoad(roadSegments);
+
+            AssertThatRoadSegmentIsCorrect(gameObjects[0], "DeadEnd(Clone)", new Vector3(2, 0, 0), Direction.East);
+            AssertThatRoadSegmentIsCorrect(gameObjects[1], "DeadEnd(Clone)", new Vector3(3, 0, 0), Direction.West);
```

Comment:
I added 2 to each x coordinate because there seems to be a calculation in the board translator that does this.